pom.xml

```diff
@@ -604,12 +597,24 @@ limitations under the License.
                                     <shadedPattern>cosmosdb_connector_shaded.org.apache.http</shadedPattern>
                                 </relocation>
                                 <relocation>
-                                    <pattern>com.google</pattern>
-                                    <shadedPattern>cosmosdb_connector_shaded.com.google</shadedPattern>
```

Comment:
are we not shading guava anymore?

Reply:
We do - the shading instructions for guava have been duplicated before. Line 365 is having the current shading instructions.